common/utils.test.js

```diff
@@ -475,4 +476,13 @@ describe('Util Tests', () => {
       });
     });
   });
+
+  describe('capitalizeFirstLetter', () => {
+    it('should capitalize the first letter of a string', () => {
+      expect(capitalizeFirstLetter('hello')).toBe('Hello');
+    });
+    it('should capitalize the first letter of a string with dutch locale', () => {
+      expect(capitalizeFirstLetter('ĳsselmeer', 'nl')).toBe('Ĳsselmeer');
+    });
+  });
```

Comment:
Can we add some test with some Japanese and RLT languages? just to be sure that this will work with those.

Reply:
Added Japanese.

It seems there is no existing RTL language that contains capital letters so should be fine.